stackable-bcrypt/src/main.rs

```diff
@@ -0,0 +1,30 @@
+use clap::Parser;
+use std::process::exit;
+
+#[derive(Parser)]
+#[clap(
+    about = "bcrypt commandline tool",
+    long_about = "A tiny command line tool to hash a given string using the bcrypt algorithm.\
+        Default cost used is 10, but can be configured. \
+        The output is only the hash, so that this can be used for automation purposes without any parsing.",
+    author = "Stackable GmbH - info@stackable.de"
+)]
+struct Opts {
+    #[clap(short, long, default_value = "10")]
+    cost: u8,
+    #[clap(short, long)]
+    input: String,
```

Comment:
Can we take this from stdin instead? argv is world-readable from /proc for the lifetime of the command.

Reply:
That wouldn't help, bash still resolves substitutions before handing off to the program. What would be an alternative would be to have the program take the path to read from as an argument.

Reply:
reads from stdin now